src/main/java/dev/dfonline/codeclient/command/impl/CommandWorldPlot.java

```diff
@@ -0,0 +1,36 @@
+package dev.dfonline.codeclient.command.impl;
+
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import dev.dfonline.codeclient.CodeClient;
+import dev.dfonline.codeclient.command.Command;
+import dev.dfonline.codeclient.location.Plot;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.literal;
+
+public class CommandWorldPlot extends Command {
+    @Override
+    public String name() {
+        return "worldplot";
```

Comment:
I would add a more sane alias, `plotsize` or `setplotsize` or `updateplotsize`, maybe we could add cc to the name. it doesn't really matter how long it is.

Reply:
plotsize is good